docs/trulens_eval/groundedness_smoke_tests.ipynb

```diff
@@ -0,0 +1 @@
+/home/daniel/git/trulens/trulens_eval/trulens_eval/tests/groundedness_smoke_tests.ipynb
```

Comment:
This looks like a local filepath. Can you fix?

Reply:
thanks for catching this! done